ee/tabby-webserver/src/service/event_logger.rs

```diff
@@ -70,8 +101,23 @@ impl EventLogger for DbEventLogger {
                         return;
                     };
                     log_err(
-                        db.create_user_completion(x.ts, user_db.id, completion_id, language)
-                            .await,
+                        db.create_user_completion(
+                            x.ts,
+                            user_db.id,
+                            completion_id.clone(),
+                            language,
+                        )
+                        .await,
+                    );
+                    log_err(
+                        db.create_user_event(
+                            user_db.id,
```

Comment:
since id is in side of `user` field, we don't need to do a lookup with `db.get_user` to ensure it exist - we can just insert the event and let foreign constraint should help us filter the problematic requests.

due to this reason, we don't really need the `create_user_event_lookup_user` method.